src/qt/sendcoinsdialog.cpp

```diff
@@ -230,8 +230,9 @@ void SendCoinsDialog::on_sendButton_clicked()
             {
                 recipients.append(entry->getValue());
             }
-            else
+            else if (valid)
```

Comment:
nit: formatting
```suggestion
            else if (valid) {
```

Reply:
This matches the style of the surrounding code and has two ACKs, so I'm going to leave it as is.